baklava/src/electron.ts

```diff
@@ -88,7 +88,7 @@ function createMainWindow() {
     mainWindow.webContents.openDevTools();
   }
   mainWindow.loadURL(
-    __prod__ ? `https://dogehouse.tv/` : "http://localhost:3000"
+    !__prod__ ? `https://dogehouse.tv/` : "http://localhost:3000"
```

Comment:
```suggestion
    __prod__ ? `https://dogehouse.tv/` : "http://localhost:3000"
```

Reply:
yea i noticed this, anyway this is still a draft cuz i had to sleep and do a few more tests